modules/platform/nuxeo-platform-document-routing/nuxeo-routing-api/src/main/java/org/nuxeo/ecm/platform/routing/api/DocumentRoutingService.java

```diff
@@ -622,4 +622,19 @@ void grantPermissionToTaskDelegatedActors(CoreSession session, String permission
      */
     boolean isWorkflowModel(final DocumentRoute documentRoute);
 
+    /**
+     * Returns true if the workflowModel can be started on the document list
+     * @param documentIds
+     * @param worflowModelName
+     * @since 11.3
```

Comment:
This will need to be updated to `@since 11.4` now everywhere.

Reply:
done